styleguide/package.json

```diff
@@ -12,7 +12,9 @@
     "build-storybook": "build-storybook",
     "build": "rimraf -rf dist && tsc",
     "depcheck": "echo '@chainlink/styleguide' && depcheck --ignore-dirs=dist || true",
-    "setup": "yarn build"
+    "setup": "yarn build",
+    "test": "jest",
+    "test:ci": "yarn test --coverage --reporters jest-silent-reporter --maxWorkers=50%"
```

Comment:
Out of curiosity, why we do use the silent reporter?

Reply:
So that there's less output when everything passes. I don't feel strongly about it.

Reply:
Maybe we should just remove the dep across the project, one less dep to worry about